releases/flying-microtonal-banana/index.md

```diff
@@ -9,11 +9,11 @@ title: "Flying Microtonal Banana"
 > Mirroring the stars are the atoms  
 > Mixing up like a cocktail
 
-Release Date: 24 February 2017
+Release Date: 24 February 2017.
 
-Volume 1 of ‘Explorations into Microtonal Tuning’. Named after a custom yellow Flying V gifted to the band leader by a guitar maker friend, the instrument was made with a banana shaped head stock and custom fret board that included extra frets for microtonal scales based on the Bağlama. This Turkish folk instrument was the fourth (and would be the last) chosen in a project to learn a new instrument every year.
+Volume 1 of ‘Explorations into Microtonal Tuning’. Named after a custom yellow Flying V gifted to Stu by a luthier friend, who made the instrument with a banana shaped head stock and custom fret board including extra frets for microtonal scales based on the Bağlama. This Turkish folk instrument was the fourth (and would be the last) chosen in a project to learn a new instrument every year.
```

Comment:
could link "Stu" to the band bio page

Reply:
could also link...
* [Flying V](https://en.wikipedia.org/wiki/Gibson_Flying_V)
* [Bağlama](https://en.wikipedia.org/wiki/Ba%C4%9Flama)